transcendence-app/src/chat/chat.controller.ts

```diff
@@ -335,7 +335,7 @@ export class ChatController {
     if (!messages) {
       throw new ServiceUnavailableException();
     }
-    return messages;
+    return messages.filter((message) => ability.can(Action.Read, message));
```

Comment:
this is not strictly necessary in our current setup, but it could be useful in the unlikely case when we wanted to implement the filtering of blocked users messages here. It could be made within the casl framework, I believe,